examples/nextjs/pages/_app.js

```diff
@@ -11,7 +11,7 @@ export default function MyApp(props) {
   React.useEffect(() => {
     // Remove the server-side injected CSS.
     const jssStyles = document.querySelector('#jss-server-side');
-    if (jssStyles) {
+    if (jssStyles && !process.browser) {
```

Comment:
This is equivalent to

```suggestion
    if (false) {
```